src/renderer/components/open-project-page/open-project-page.tsx

```diff
@@ -38,6 +46,20 @@ export const OpenProjectPage: FC<{ onNext: Consumer<void>; onBack: MouseEventHan
     }, [error]);
   }
 
+  const useResumeCheck = (onResumeStatus: Consumer<void>) => {
+    const { t } = useTranslation();
+    useEffectOnce(() => {
+      const reservedStateValues = RETAINED_LOCALSTORAGE_KEYS.map((state) => localStorage.getItem(state));
+      if (!reservedStateValues.includes(null)) {
+        if (confirm(`${t('Do you want to resume your previous records?')}`)) {
```

Comment:
What happens if the user says no? Should you not clear these values so they don't get asked again?